docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -2,9 +2,32 @@
 
 Apache Pekko 1.1.x releases support Java 8 and above.
 
+## 1.1.3
+
+Pekko 1.1.3 is a bug fix and dependency upgrade release. See [GitHub Milestone for 1.1.3](https://github.com/apache/pekko/milestone/13?closed=1) for a fuller list of changes.
```

Comment:
Should we backport https://github.com/apache/pekko/pull/1505 too?

Reply:
RC1 vote has already started

Reply:
https://github.com/apache/pekko/pull/1611

This is not been ported to 1.1.3